src/devices/Ssd1331/Ssd1331/Ssd1331Constants.cs

```diff
@@ -0,0 +1,157 @@
+            { 0x7F,0x04,0x08,0x10,0x7F,0x00 } , /* N  */
+            { 0x3E,0x41,0x41,0x41,0x3E,0x00 } , /* O  */
+            { 0x7F,0x09,0x09,0x09,0x06,0x00 } , /* P  */
+            { 0x3E,0x41,0x51,0x21,0x5E,0x00 } , /* Q  */
+            { 0x7F,0x09,0x19,0x29,0x46,0x00 } , /* R  */
+            { 0x26,0x49,0x49,0x49,0x32,0x00 } , /* S  */
+            { 0x01,0x01,0x7F,0x01,0x01,0x00 } , /* T  */
+            { 0x3F,0x40,0x40,0x40,0x3F,0x00 } , /* U  */
+            { 0x03,0x1C,0x60,0x1C,0x03,0x00 } , /* V  */
+            { 0x0F,0x70,0x0F,0x70,0x0F,0x00 } , /* W  */
+            { 0x41,0x36,0x08,0x36,0x41,0x00 } , /* X  */
+            { 0x01,0x06,0x78,0x02,0x01,0x00 } , /* Y  */
+            { 0x61,0x51,0x49,0x45,0x43,0x00 } , /* Z  */
+            { 0x00,0x00,0x7F,0x41,0x41,0x00 } , /* [  */
+            { 0x15,0x16,0x7C,0x16,0x11,0x00 } , /* \  */
+            { 0x41,0x41,0x7F,0x00,0x00,0x00 } , /* ]  */
+            { 0x00,0x02,0x01,0x02,0x00,0x00 } , /* ^  */
+            { 0x40,0x40,0x40,0x40,0x40,0x00 } , /* _  */
+            { 0x00,0x01,0x02,0x00,0x00,0x00 } , /* `  */
+            { 0x00,0x20,0x54,0x54,0x78,0x00 } , /* a  */
+            { 0x00,0x7F,0x44,0x44,0x38,0x00 } , /* b  */
+            { 0x00,0x38,0x44,0x44,0x28,0x00 } , /* c  */
+            { 0x00,0x38,0x44,0x44,0x7F,0x00 } , /* d  */
+            { 0x00,0x38,0x54,0x54,0x18,0x00 } , /* e  */
+            { 0x00,0x04,0x3E,0x05,0x01,0x00 } , /* f  */
+            { 0x00,0x08,0x54,0x54,0x3C,0x00 } , /* g  */
+            { 0x00,0x7F,0x04,0x04,0x78,0x00 } , /* h  */
+            { 0x00,0x00,0x7D,0x00,0x00,0x00 } , /* i  */
+            { 0x00,0x40,0x40,0x3D,0x00,0x00 } , /* j  */
+            { 0x00,0x7F,0x10,0x28,0x44,0x00 } , /* k  */
+            { 0x00,0x01,0x7F,0x00,0x00,0x00 } , /* l  */
+            { 0x7C,0x04,0x7C,0x04,0x78,0x00 } , /* m  */
+            { 0x00,0x7C,0x04,0x04,0x78,0x00 } , /* n  */
+            { 0x00,0x38,0x44,0x44,0x38,0x00 } , /* o  */
+            { 0x00,0x7C,0x14,0x14,0x08,0x00 } , /* p  */
+            { 0x00,0x08,0x14,0x14,0x7C,0x00 } , /* q  */
+            { 0x00,0x7C,0x08,0x04,0x04,0x00 } , /* r  */
+            { 0x00,0x48,0x54,0x54,0x24,0x00 } , /* s  */
+            { 0x00,0x04,0x3E,0x44,0x40,0x00 } , /* t  */
+            { 0x00,0x3C,0x40,0x40,0x7C,0x00 } , /* u  */
+            { 0x00,0x7C,0x20,0x10,0x0C,0x00 } , /* v  */
+            { 0x1C,0x60,0x1C,0x60,0x1C,0x00 } , /* w  */
+            { 0x00,0x6C,0x10,0x10,0x6C,0x00 } , /* x  */
+            { 0x00,0x4C,0x50,0x30,0x1C,0x00 } , /* y  */
+            { 0x00,0x44,0x64,0x54,0x4C,0x00 } , /* z  */
+            { 0x00,0x08,0x36,0x41,0x41,0x00 } , /* {  */
+            { 0x00,0x00,0x7F,0x00,0x00,0x00 } , /* |  */
+            { 0x41,0x41,0x36,0x08,0x00,0x00 } , /* }  */
+            { 0x08,0x04,0x08,0x10,0x08,0x00 } , /* ~  */
+            { 0x00,0x00,0x00,0x00,0x00,0x00 }   /*null*/
+        };
+
+        private const byte Ssd1331_SetColumnAddress = 0x15;

```

Comment:
Please see the remarks on const and variable naming, we're not using underscore in the name